### PR TITLE
[ticket/14095] Pure CSS loader...No Image

### PR DESCRIPTION
**Pull request by hanakin**
Checklist:

- [x] Correct branch: master for 3.3
- [x] Tests pass
- [x] Code follows coding guidelines
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

PHPBB3-14095

https://tracker.phpbb.com/browse/PHPBB3-14095

### CODE REVIEW

File: phpBB/styles/prosilver/template/simple_footer.html [hunk 1]
```diff
@@ -8,8 +8,9 @@
 	<div id="darkenwrapper" class="darkenwrapper" data-ajax-error-title="{L_AJAX_ERROR_TITLE}" data-ajax-error-text="{L_AJAX_ERROR_TEXT}" data-ajax-error-text-abort="{L_AJAX_ERROR_TEXT_ABORT}" data-ajax-error-text-timeout="{L_AJAX_ERROR_TEXT_TIMEOUT}" data-ajax-error-text-parsererror="{L_AJAX_ERROR_TEXT_PARSERERROR}">
 		<div id="darken" class="darken">&nbsp;</div>
 	</div>
-	<div id="loading_indicator" class="loading_indicator"></div>
-
+	<div id="loading_indicator" class="loading_indicator">
+		<div class="loader"></div>
```
Comment (Crizz0) [inline phpBB/styles/prosilver/template/simple_footer.html]: Do we need an alt text for the loading image/thing? Empty elements are always so ugly :/

**Comment by hanakin** [comment]
!set Styles

**Comment by hanakin** [comment]
sure thing its already hidden from none screen readers anyway

**Comment by Crizz0** [comment]
Well, I think a language variable would be better then.

**Comment by VSEphpbb** [comment]
Interesting PR title. Any pics of what they look like?

**Comment by hanakin** [comment]
screenshot 2017-02-17 18 21 46

**Comment by hanakin** [comment]
@Louis7777 

perhaps something like this would work better its faster loading and does not require darken wrapper

example in action http://codepen.io/hanakin/pen/aprEZB

**Comment by Louis7777** [comment]
I think it looks annoying. Both the animation and the colors.

If you'd like to get rid of the dark wrapper then how about something like the "Bars" or the "LoadBar":

http://codepen.io/bernethe/pen/dorozd

Or anything more soothing than what seems like a police siren light. :P

**Comment by hanakin** [comment]
@Louis7777 ignore the colors as those are just prosilvers base colors this is just a simplified version of gmails loader with two color swap, im not apposed to keeping the dark block, im just trying to make it easier to swap out other loaders. As well as add some custimization options

there are really hundreds of options

http://codepen.io/Iulius90/pen/RaeWmY?editors=1100

**Comment by Louis7777** [comment]
> http://codepen.io/Iulius90/pen/RaeWmY?editors=1100

Maybe the red fat spinner would be appropriate (2nd row, 3rd column)?

> http://cssload.net/en/spinners

First page, first row, one of the last three (preferably the middle one)?

**Comment by hanakin** [comment]
perhaps something utilizing an svg animation https://codepen.io/mrrocks/pen/EiplA

**Comment by Louis7777** [comment]
I think it's fine both as an implementation and as an animation.

**Comment by hanakin** [comment]
updated to gmd spinner

example: http://codepen.io/hanakin/full/aprEZB/

Screen:
screenshot 2017-02-23 13 34 39

**Comment by marc1706** [comment]
AJAX operations in the ACP still seem to use the old spinner.

**Comment by hanakin** [comment]
@marc1706 have not touched ACP fully yet

**Comment by hanakin** [comment]
@marc1706 Should be fixed now

**Comment by marc1706** [comment]
Thanks, will take a look once I have some spare time.

**Pull request merged**